tests/foreman/ui/test_org.py

```diff
@@ -276,15 +275,9 @@ def test_positive_delete_bz1225588(self):
             # Org cannot be deleted when selected,
             # So switching to Default Org and then deleting.
             session.nav.go_to_select_org('Default Organization')
-            self.org.remove(org_name)
+            self.org.delete(org_name)
             session.nav.go_to_dashboard()
-            status = self.org.search(org_name)
-            # Check for at least ten times that org is deleted due #1225588
-            for _ in range(10):
-                status = self.org.search(org_name)
-                if status is None:
-                    break
-            self.assertIsNone(status)
+            self.assertIsNone(self.org.search(org_name))
```

Comment:
the previous test is not asserting after delete, but this one does.  Can we follow one approach?